corelib-schemaorg/src/test/java/eu/europeana/corelib/edm/utils/SchemaOrgUtilsTest.java

```diff
@@ -45,6 +50,7 @@ public void toSchemaOrgTest() throws IOException, URISyntaxException {
         String expectedOutput = IOUtils.toString(stream, StandardCharsets.UTF_8);
         //we cannot string compare until the ordering of properties is implemented
         //still, a fast indication that the output was changed will be indicated through the length of the string
+        System.out.println(output);
```

Comment:
Please do not use System.out

Reply:
I can reformat the whole class, as it's an already existing class and I just followed the same indent